charts/karpenter-crd/templates/karpenter.sh_nodeclaims.yaml

```diff
@@ -380,440 +389,6 @@ spec:
       storage: true
       subresources:
         status: {}
-    - additionalPrinterColumns:
```

Comment:
Originally brought up by @charliedmcb: What notice have we given users that this break is coming?

Reply:
That's a good point. Do we normally consider print columns part of public API?

Reply:
Sorry I put this comment in an awkward place: I meant the v1beta1 removal.